views/main.html

```diff
@@ -327,7 +327,7 @@
   <div class="modal fade" tabindex="-1" role="dialog" id="tagModal">
     <div style="position:fixed;left:0;top:0">
       <span ng-repeat="shot in filmDB[fetchedTagsForFilm].sshot">
-        <img src="{{shot}}" class="img-responsive" style="height:20vh;width:auto;">
+        <img ng-src={{shot}}" class="img-responsive" style="height:20vh;width:auto;">
```

Comment:
Page doesn't try to load "{{shot}}" before angular bindings happen.
fixes the console error about %78%78shot%7B%7B not found on server.